packages/core/data-transfer/src/strapi/providers/remote-destination/index.ts

```diff
@@ -78,33 +101,48 @@ class RemoteStrapiDestinationProvider implements IDestinationProvider {
       return new ProviderTransferError('Unexpected error');
     }
 
+    this.stats[step] = { count: 0 };
+
     return null;
   }
 
   async #endStep<T extends Client.TransferPushStep>(step: T) {
     try {
-      await this.dispatcher?.dispatchTransferStep({ action: 'end', step });
+      const res = await this.dispatcher?.dispatchTransferStep<{
+        ok: boolean;
+        stats: { started: number; finished: number };
```

Comment:
We're using the `{ started: number; finished: number }` type in a lot of places, it might be worth creating a dedicated type for it, what do you think?